packages/contracts/contracts/voting/majority/IMajorityVoting.sol

```diff
@@ -56,30 +85,30 @@ interface IMajorityVoting {
     event ProposalExecuted(uint256 indexed proposalId, bytes[] execResults);
 
     /// @notice Emitted when the vote settings are updated.
-    /// @param relativeSupportThresholdPct The support threshold in percent.
-    /// @param totalSupportThresholdPct The total support threshold in percent.
+    /// @param supportThreshold The support threshold in percent.
+    /// @param participationThreshold The participation threshold in percent.
     /// @param minDuration The minimal duration of a vote.
     event VoteSettingsUpdated(
-        uint64 totalSupportThresholdPct,
-        uint64 relativeSupportThresholdPct,
+        uint64 participationThreshold,
+        uint64 supportThreshold,
         uint64 minDuration
     );
 
     /// @notice Changes the vote settings.
-    /// @param _totalSupportThresholdPct The total support threshold in percent.
-    /// @param _relativeSupportThresholdPct The relative support threshold in percent.
+    /// @param _participationThreshold The participation threshold in percent.
+    /// @param _supportThreshold The support threshold in percent.
     /// @param _minDuration The minimal duration of a vote.
     function changeVoteSettings(
-        uint64 _relativeSupportThresholdPct,
-        uint64 _totalSupportThresholdPct,
+        uint64 _supportThreshold,
+        uint64 _participationThreshold,
```

Comment:
It may not be a big deal, but in some cases above, the order of the parameters was inverted. I would suggest always passing them in the very same order... to avoid unexpected mismatches

Reply:
Fixed globally in 594260c92b86f87970a9060d9e8700272d77cfdd